code/modules/unit_tests/map_tests.dm

```diff
@@ -145,16 +81,6 @@
 	Fail(origin_turf, "tile has an unconnected cable ([report_name] connection: [uppertext(dir2text(direction))]).")
 	return FALSE
 
-/datum/map_per_tile_test/duplicate_cable_check
-
-/datum/map_per_tile_test/duplicate_cable_check/CheckTile(turf/T)
-	var/obj/structure/cable/cable = locate() in T.contents
-	for(var/obj/structure/cable/other_cable in T.contents)
-		if(cable == other_cable)
-			continue // same object, continue
-		if(cable.d1 == other_cable.d1 && cable.d2 == other_cable.d2)
-			Fail(T, "tile has duplicated cables.")
```

Comment:
accidental removal? These need to check that they have the exact same direction so a normal maplint wont work here

Reply:
This and the other cable test are both already covered by `stacked_matching_cables.yml`:
```yaml
/obj/structure/cable:
  banned_neighbors:
    /obj/structure/cable:
      identical: true
```

This prevents all cables with the same varedits. So if two cables have the same icon-state, they'll fail this lint.

Reply:
oh, amazing